src/main/java/ai/zuva/docai/normalization/NormalizationRequest.java

```diff
@@ -0,0 +1,17 @@
+package ai.zuva.docai.normalization;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class NormalizationRequest {
```

Comment:
I think we can just call this class `Normalization` and have the static methods `GetDate`, `GetDuration` and `GetCurrency`. 

The `createRequest` methods don't really make sense, because they directly return results, not requests, and the `NormalizationRequest` class and its descendants aren't really adding much value here.

Reply:
827385e0eb4ae8c753b61e4cce8e00174d27d8d1